src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -4,42 +4,57 @@ const HeaderUserMenu = (props) => {
   const user = props.user;
   return (
     <div>
-      <div>
-        <span style={{ background: user.avatar.color }}>{user.avatar.text}</span>
-        <div>
-          <span>{user.fullname}</span>
-          <span>{user.email}</span>
-        </div>
-      </div>
-      <div>
-        <div>
-          {user.plan.isSubscribers === 'true' || user.plan.isMonthlyByEmail === 'true' ? (
+      <span id="user-menu--open">
+        <span style={{ background: user.avatar.color }} className="user-avatar">
+          {user.avatar.text}
+        </span>
+      </span>
+      <div class="user-menu">
+        <header>
+          <span className="user-avatar--menu" style={{ background: user.avatar.color }}>
```

Comment:
Here you have an extra space `style={{ background: user.avatar.color }}` but in another properties you don't, this is only for style property? See below `{user.avatar.text}`

Reply:
Fixed